packages/atlas/scripts/customSchemaLoader.ts

```diff
@@ -17,30 +17,34 @@ async function fetchAuthCookie() {
   return response.headers['set-cookie']
 }
 
-export async function customSchemaLoader() {
-  const authCookie = await fetchAuthCookie()
-  const introspectionQuery = getIntrospectionQuery()
+export function customSchemaLoader(schemaUrl: string, authUrl: string) {
+  return async () => {
+    const authCookie = await fetchAuthCookie(authUrl)
```

Comment:
isn't the schema available without the auth cookie now?

Reply:
Yeah probably actually I just copied the code I added on the notification branch a while ago